Objects/dictobject.c

```diff
@@ -300,18 +300,26 @@ _PyDict_DebugMallocStats(FILE *out)
 
 static void free_keys_object(PyInterpreterState *interp, PyDictKeysObject *keys);
 
+// XXX Switch to Py_INCREF()?
```

Comment:
Right, any switch would require a slight change to the `PyDictObject` layout, but only if `Py_TRACE_REFS` is defined.  `PyObject.ob_refcnt` is otherwise at the beginning of the struct, so the rest wouldn't matter.

That said, the only reason why it would matter is to get the performance benefits of `Py_INCREF()` for immortal objects.  In the case of `Py_EMPTY_KEYS` I expect the benefit would be negligible, so it mostly wouldn't be worth the trouble.

Normally I'd clarify the comment, but I'll probably drop it instead.

Reply:
done